.vscode/launch.json

```diff
@@ -15,7 +15,7 @@
       "name": "Launch",
       "type": "lldb",
       "request": "launch",
-      "program": "~/Library/Developer/Xcode/DerivedData/Demo-avuzscipzqxczrbltxhlvbnxujdo/Build/Products/Debug-${command:ios-debug.targetSdk}/Demo.app",
+      "program": "~/Library/Developer/Xcode/DerivedData/Demo/Build/Products/Debug-${command:ios-debug.targetSdk}/Demo.app",
```

Comment:
Was looking into passing Xcode project has as an argument:
https://github.com/Microsoft/vscode/issues/69027

Reply:
started making a command line tool to generate hash:
https://github.com/markst/xcode-project-hashes

Reply:
does seem like best idea is to go about creating an extension: https://github.com/markst/hotreloading-vscode-ios/issues/4